payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -51,18 +60,100 @@ internal abstract class BaseAddPaymentMethodFragment(
         val viewBinding = FragmentPaymentsheetAddPaymentMethodBinding.bind(view)
         addPaymentMethodHeader = viewBinding.addPaymentMethodHeader
 
-        addPaymentMethodFragment()
+        val config = requireNotNull(
+            arguments?.getParcelable<FragmentConfig>(
+                BaseSheetActivity.EXTRA_FRAGMENT_CONFIG
+            )
+        )
+
+        val paymentMethods = config.stripeIntent.paymentMethodTypes.mapNotNull {
+            SupportedPaymentMethod.fromCode(it)
+        }.filter { it == SupportedPaymentMethod.Card }
```

Comment:
Removing this filter and using the EUR currency in the playground lets you test the selector.

Reply:
Does this filter have a longer standing need, or just make it so we can merge to master without actually exposing this :D

Reply:
The Ideal form right now is fake, it only exists because I needed a way to run this implementation. I'm coordinating with Michelle to get both our changes in so we can integrate them and use the real forms that are in the `paymentsheet` module. This hack won't be long lived.